components/eam/src/chemistry/yaml/mo_chm_diags/chm_diags_output_ts_355.py

```diff
@@ -0,0 +1,79 @@
+input.fldcw_nn40=[[  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,  0.00000000000000000E+000,],]
+# Output data is stored here.
+output = Object()
+output.area=[[  0.11857287250233438E+013],]
```

Comment:
In the fortran this is populated by a call to `get_area_all_p` - is this another instance where this value will come from the host model on the C++ side?

Reply:
not quite understand this question. @singhbalwinder can you help on it?

Reply:
Like for `het_diags` how `wght` is now provided instead of calling `get_wght_all_p`

Reply:
I called `get_wght_all_p` before writing out wght.  see   lines 95-96 in https://github.com/eagles-project/e3sm_mam4_refactor/pull/149/files#diff-9acb5cef1c6ffae77d0e65484acc4fda0e5d89e84e6090d37ce3bf671b75720eR95 

when you do testing, you may comment out  `get_wght_all_p` and just uses the input file. I am not sure how you will refactor this call in C++.

Reply:
You are right @jaelynlitz, `get_area_all_p` will come from the host model. Thanks @tangshuaiqi for printing this variable in the validation data. On the C++ side, please hardwire a value for now but add a comment that it should come from the host model. We should collect all these variables somewhere so that we don't forget about them.

Reply:
Gotcha, I understand, thanks!